core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -113,7 +115,7 @@ class DAGScheduler(
   private val failedEpoch = new HashMap[String, Long]
 
   private val dagSchedulerActorSupervisor =
-    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)))
+    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)), "DAGSchedulerSupervisor")
```

Comment:
Hmm.  Maybe we could just start up a new actor that acts as an endpoint for receiving requests from the executors and forwarding them to DAGScheduler for further processing.  I considered making OutputCommitCoordinator into an actor, so maybe that could work.